src/cloud/main.js

```diff
@@ -102,3 +102,20 @@ Parse.Cloud.define('requiredParameterCheck', function(req, res) {
 }, function(params) {
   return params.name;
 });
+
+Parse.Cloud.define("PromiseWithoutRejectedCallbackFail", function(request, response) {
+  var promises = [];
+	var result = {};
+
+	// fetch config
+	result.config = {};
+
+	promises.push(Parse.Config.get().then(function(config) {
+		result.config = config.attributes;
```

Comment:
if you add `return Parse.Promise.as()` there, does it changes any thing?

Can you add a Unit test as well so the failure is properly handled and we ensure no regression in the future?